### DISCUSSION TITLE
Is the \.github folder for github only or should I put supporting code in there.

### DISCUSSION THREAD

**Discussion by flagrant99** [community/community#68423] [Actions] [answered]
### Select Topic Area

Question

### Body

I have a bunch of yml in \.github\workflows, These call supporting powershell code files. Is it OK to make my own folder under .github to store and organize supporting powershell code or should I keep my hands off .github folder. Is that for github only? I mean I guess I could create a _github folder for my own supporting code.

**Comment by yashSingh97** [comment, marked as answer]
While .github is meant for github specifc configs only, it shouldn't matter if you decide to make your own folders within it. Another way to look at it is, the way one organizes the repo shows one's professionalism, so using a folder like utils or scripts might be better, but it should be fine either way.
